server.go

```diff
@@ -514,8 +523,9 @@ func (s *Server) HandleGetHeader(w http.ResponseWriter, r *http.Request) {
 	validatorID := r.Context().Value(keyOrgID).(string)
 	authHeader := r.Context().Value(keyAuthHeader).(string)
 	accountID := r.Context().Value(keyAccountID).(string)
-	getHeaderStartTimeUnixMS := r.Header.Get(MEVBoostStartTimeUnixMS)
+	mevBoostSendTimeUnixMS := r.Header.Get(MEVBoostStartTimeUnixMS)
 	commitBoostSendTimeUnixMS := r.Header.Get(commitBoostStartTimeUnixMS)
```

Comment:
can we rename this to `HeaderDateMilliseconds` to align with mev-boost?

Reply:
same with the constant

Reply:
MEV Boost is gonna keep both for now. we can change it to a single field later